src/setupTests.ts

```diff
@@ -0,0 +1,8 @@
+import "@testing-library/jest-dom/vitest"
+import { afterEach} from 'vitest'
+import { cleanup } from '@testing-library/react'
+
+// runs a clean after each test case (e.g. clearing jsdom)
```

Comment:
remove comments that are not needed for now, you can clarify this in the Readme if necessary.

Reply:
Resolved.